apps/e2e-tests/tests/checkCategory.spec.ts

```diff
@@ -39,8 +39,9 @@ test.describe.only("Check category page", () => {
   test("Check sorting", async ({ page }) => {
     await homePage.openCategoryPage();
     await categoryPage.selectSortingPriceAsc();
-    await expect(page.getByTestId("loading")).toHaveCount(0);
+    await page.waitForLoadState();
```

Comment:
do we need to call it for a second time as it had been executed once before within `categoryPage.selectSortingPriceAsc();` ?

Reply:
you are right, it is unnecessary